src/pages/WalletManagement/ImportWordsPage.tsx

```diff
@@ -84,7 +82,8 @@ const ImportWordsPage = () => {
     }
   }
 
-  const isNextButtonActive = phrase.length === 24
+  // Alephium's node code uses 12 as the minimal mnemomic length.
```

Comment:
Shall we put a link to the scala file? Or do you think it's unnecessary?

Reply:
Nah, while I doubt it, it's possible the file could be renamed, moved, or the content in the file no longer refers to what we mentioned.